tilewe/engine.py

```diff
@@ -33,22 +34,27 @@ def on_search(self, board: tilewe.Board, _seconds: float) -> tilewe.Move:
 
 class Tournament: 
 
-    def __init__(self, engines: list[Engine]): 
-        self.engines = list(engines) 
+    def __init__(self, engines: list[Engine], move_seconds: int=60): 
+        self.engines = list(engines)
+        self.move_seconds = move_seconds
 
-    def play(self, n_games: int, n_threads: int=1): 
+    def play(self, n_games: int, n_threads: int=1, move_seconds: int=None):
+        # initialize trackers and game controls
         games = 0
         wins = [0 for _ in range(len(self.engines))]
         totals = [0 for _ in range(len(self.engines))]
+        if move_seconds is not None: self.move_seconds = move_seconds
```

Comment:
Would it not make more sense for `move_seconds` to override the time control for only this set of matches so that the default tournament settings are kept for later ones?

Reply:
I think it would, yeah

Reply:
That's a good 10am vs 2am thought